openfisca_us/tests/policy/baseline/hhs/ccdf/income_to_smi_ratio.yaml

```diff
@@ -32,4 +32,4 @@
     spm_unit_size: 7
     state_code_str: NY
   output:
-    ccdf_income_to_smi_ratio: 1.35 * 100_000 / 104_972
+    ccdf_income_to_smi_ratio: 100_000 / (104_972 * 1.35)
```

Comment:
Nit for consistency
```suggestion
    ccdf_income_to_smi_ratio: 100_000 / (1.35 * 104_972)
```

Reply:
I changed these to the calculated values for hhs_smi in the latest commit. Would that be ok?
- name: Household size 7, State NY
  period: 2022
  absolute_error_margin: 0.01
  input:
    ccdf_income: 100_000
    hhs_smi: 141_712.2
  output:
    ccdf_income_to_smi_ratio: 100_000 / 141_712.2
All previous conversations have already been resolved.

Reply:
Sounds good, could you please click `Resolve conversation` on each? This helps indicate that they've been addressed.